internal/scm/github/github.go

```diff
@@ -221,23 +228,24 @@ func (g *Github) getRepositories(ctx context.Context) ([]*github.Repository, err
 		}
 		allRepos = append(allRepos, repos...)
 	}
-
-	// Remove duplicate repos
-	repoMap := map[string]*github.Repository{}
-	for _, repo := range allRepos {
-		repoMap[repo.GetFullName()] = repo
-	}
-	allRepos = make([]*github.Repository, 0, len(repoMap))
-	for _, repo := range repoMap {
-		if repo.GetArchived() || repo.GetDisabled() {
-			continue
-		}
-		allRepos = append(allRepos, repo)
-	}
+	filteredRepos := make([]*github.Repository, 0, len(allRepos))
+	// Filter repositories
+	filteredRepos = slices.DeleteFunc(allRepos, func(repo *github.Repository) bool {
```

Comment:
```suggestion
	// Filter repositories
	filteredRepos := slices.DeleteFunc(allRepos, func(repo *github.Repository) bool {
```

The initial creation with `make` is thrown away immediately when `DeleteFunc` returns: `DeleteFunc` will return a slice it makes from `allRepos`, so it isn't necessary to allocate `filteredRepos` separately.